docs/src/extend/plugin-migration-flat-config.md

```diff
@@ -220,7 +219,9 @@ module.exports = {
 };
 ```
 
-You should update your documentation so your plugin users know how to reference the exported configs. If your config is an array, consumers will have to spread it:
+You should update your documentation so your plugin users know how to reference the exported configs.
+
+If your config is an object, consumers should not spread it; and if it's an array, they should:
```

Comment:
This still implies that the following example shows just how to spread, and the verbiage is more confusing. Maybe something like this?
```suggestion
If your exported config is an object, then your users can insert it directly into the config array; if your exported config is an array, then your users should use the spread operator (`...`) to insert the array's items into the config array. Here's an example with both an object config and an array config:
```